geo.txt

```diff
@@ -56,3 +56,731 @@ Phys 91(11):957-962
 # Date first added: 2023-05-15 (by Nike Dattani)
 
 H2 Ar 3.957
+
+NAME = AgClH3N  2010
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Mikhailov VA, Tew DP, Walker NR, Legon AC (2010) H3N⋅⋅⋅Ag–Cl: Synthesis in a supersonic jet and
+characterization by rotational spectroscopy. Chem Phys Lett 499(1-3):16-20
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Ag   Cl     2.26333(6)
+Ag...N      2.15444(6)
+N    H      1.0129
+H    N...Ag 113.48(2)
+
+# Bond lengths: rs     (angstroms)
+
+Ag   Cl     2.2633(9)
+Ag...N      2.1545(8)
+
+NAME = AgH3IN
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Medcraft C, Gougoula E, Bittner DM, Mullaney JC, Blanco S, Tew DP, Walker NR, Legon AC (2017)
+Molecular geometries and other properties of H2O⋅⋅⋅AgI and H3N⋅⋅⋅AgI as characterized by rotational
+spectroscopy and ab initio calculations. J Chem Phys 147(23):234308/1-234308/8
+https://doi.org/10.1063/1.5008744
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Ag   I   2.5375(3)
+Ag...N   2.180(1)
+Ag   N H 110.86(5)
+
+# Bond lengths: rs     (angstroms)
+# Bond angles:  thetas (degrees)
+
+Ag...N   2.182(1)
+Ag   N H 110.93(3)
+
+NAME = AgH3IP
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Stephens SL, Tew DP, Walker NR, Legon AC (2016) H3P⋅⋅⋅AgI: generation by laser-ablation and
+characterization by rotational spectroscopy and ab initio calculations. Phys Chem Chem Phys 18(28):18971-
+18977
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Ag  I   2.5483(1)
+P...Ag  2.3488(20)
+P   H   1.4086
+H   P H 118.92 
```

Comment:
The book has a footnote on the number 118.92. The footnote says "Assumed". I think it's important for our database to retain the information that 118.92 is an "assumed" angle.

Reply:
```
Ag  I   2.5483(1)
P...Ag  2.3488(20)
P   H   1.4086
H   P H 118.92(b)

(b) assumed
```

Reply:
[ ] This comment can be moved up to where the rest of the comments/meta-data are. 
[ ] "H P H" can be "HPH or H-P-H". 
[ ] 118.92 doesn't need to be repeated in the comment.
[ ] assumed can be put in inverted commas: "assumed".